src/Field/Field.tsx

```diff
@@ -0,0 +1,141 @@
+import React from 'react'
+import styled from 'styled-components'
+import { darken } from 'polished'
+
+import { Icon } from '../Icon'
+import { Text } from '../Text'
+import { Box } from '../Box'
+import { theme } from '../theme'
+
+type Props = {
+  children: React.ReactElement
+  showCaret?: boolean
+  renderAsTitle?: boolean
+  className?: string
+  id: string
+  error?: boolean // nb this needs to be optional for legacy support - but we will want to enforce in  the future
+  label?: string
+  outlined?: boolean
+  value: string
+  trailingIcon?: string
+  errorMsg?: string
+  dropdownKey?: string // nb only used by dropdown component but required on content in that context
+  required?: boolean
+  fullHeight?: boolean // used to make text areas full size
+}
+
+export const Field = ({
+  children,
+  renderAsTitle,
+  className = '',
+  error,
+  id,
+  label,
+  outlined = false,
+  value,
+  trailingIcon,
+  errorMsg,
+  dropdownKey,
+  required,
+  showCaret,
+  fullHeight = false,
+}: Props) => {
+  return (
+    <Container className={className}>
+      {label && (
+        <Box mb={outlined ? '4px' : '0px'}>
+          {renderAsTitle ? (
+            <Title>{label}</Title>
+          ) : (
+            <Text tag="label" color="subtext" typo="label" htmlFor={id}>
+              {label}
+            </Text>
+          )}
+          {required && (
+            <Text tag="label" color="error" typo="label">
+              *
+            </Text>
+          )}
```

Comment:
wondering if we should move this inside the above Text and have it a span?
like
```
<Text tag="label" color="subtext" typo="label" htmlFor={id}>
{label}
{required && (
<Text tag="span" color="error" typo="label">
 *
</Text>
)}
</Text>
```

cause like this we are rendering two labels that's not ideal

Reply:
It still wants to be in a `Text` tag so that we can apply the right colour, but I've changed the `typo` to base so that we're not creating multiple labels, in [e377d0](https://github.com/marshmallow-insurance/smores-react/pull/2023/commits/e377d04ddbca32b14a683c40ee60eb58a5e54dcf)